tests/http/test_request_normalization.py

```diff
@@ -2,11 +2,12 @@
 import urllib
 
 import six
+from six.moves import urllib_parse as urlparse
 
 import requests
 
 from flex.http import (
-    normalize_request, _tornado_available, _falcon_available, _webob_available
+    normalize_request, _tornado_available, _falcon_available, _webob_available, _django_available
```

Comment:
Absolutely happy to do that, but the maximum line length given to flake8 in tox.ini is 100 chars, this is only 97 - is 100 not being enforced?